app/src/main/java/org/oppia/app/utility/FontScaleConfigurationUtil.kt

```diff
@@ -0,0 +1,35 @@
+package org.oppia.app.utility
+
+import android.content.Context
+import android.content.Context.WINDOW_SERVICE
+import android.util.DisplayMetrics
+import android.view.WindowManager
+import org.oppia.app.model.StoryTextSize
+import javax.inject.Inject
+
+/** FontScaleConfigurationUtil helps to update font scale. */
+class FontScaleConfigurationUtil @Inject constructor(
+  private val context: Context, private val storyTextSize: String
+) {
+  /** This method updates font scale. */
+  fun adjustFontScale() {
+    val configuration = context.resources.configuration
+    configuration.fontScale = getFontScaleByStoryTextSize()
+    val metrics: DisplayMetrics = context.resources.displayMetrics
+    val windowManager = context.getSystemService(WINDOW_SERVICE) as WindowManager?
```

Comment:
Prefer: ``context.getSystemService(WINDOW_SERVICE) as? WindowManager``

This is actually a safer way to cast with nullability. See: https://kotlinlang.org/docs/reference/typecasts.html#safe-nullable-cast-operator.

Reply:
Done.